testutil/compose/define.go

```diff
@@ -252,7 +252,7 @@ func buildLocal(ctx context.Context, dir string) error {
 
 	log.Info(ctx, "Building local charon binary", z.Str("repo", repo), z.Str("target", target))
 
-	cmd := exec.CommandContext(ctx, "go", "build", "-o", target)
+	cmd := exec.CommandContext(ctx, "go", "build", "-ldflags='-linkmode 'external' -extldflags '-static'", "-o", target)
```

Comment:
@gsora we are getting these errors in compose-tests:
```
[36mnode0_1  |[0m /compose/charon: /lib/x86_64-linux-gnu/libc.so.6: version `GLIBC_2.32' not found (required by /compose/charon)
[36mnode0_1  |[0m /compose/charon: /lib/x86_64-linux-gnu/libc.so.6: version `GLIBC_2.34' not found (required by /compose/charon)
```
This is why all compose-tests are failing.

Reply:
I'm aware of this situation.

It's weird because adding the specific LDFLAGS to `go build` fixes that exact error on my machine.

I'll investigate.

Reply:
Solved!